Tests/Shared.Specs/AsyncFunctionExceptionAssertionSpecs.cs

```diff
@@ -735,6 +989,29 @@ public void When_async_method_of_T_succeeds_and_expected_not_to_throw_particular
             action.Should().NotThrow();
         }
 
+#if NETCOREAPP2_0 || NETCOREAPP2_1 || NETCOREAPP3_0 || NETSTANDARD2_1
```

Comment:
Is the ` || NETSTANDARD2_1` for automatically including potentially future test projects?

Reply:
Shouldn't be there. Fixed.